src/toolbar.c

```diff
@@ -135,163 +130,158 @@ toolbar_append_stock_button (GtkToolbar *toolbar,
-																 _("Activity Journal"),
-																 _("View and Edit Timestamp Logs"),
-																 GNOME_STOCK_BOOK_OPEN,
-																 (GtkSignalFunc) show_report, ACTIVITY_REPORT);
-			position ++;
+			mytbar->journal_button = toolbar_append_stock_button (
+			    mytbar->tbar, _ ("Activity Journal"), _ ("View and Edit Timestamp Logs"),
+			    GNOME_STOCK_BOOK_OPEN, (GtkSignalFunc) show_report, ACTIVITY_REPORT
+			);
+			position++;
 		}
 		if (config_show_tb_prop)
 		{
-			mytbar->prop_w = gtk_toolbar_insert_stock (mytbar->tbar,
-													   GTK_STOCK_PROPERTIES,
-													   _("Edit Project Properties..."), NULL,
-													   (GtkSignalFunc) menu_properties, NULL,
-													   position ++);
+			mytbar->prop_w = gtk_toolbar_insert_stock (
+			    mytbar->tbar, GTK_STOCK_PROPERTIES, _ ("Edit Project Properties..."), NULL,
+			    (GtkSignalFunc) menu_properties, NULL, position++
+			);
 		}
 		if (config_show_tb_timer)
 		{
 			/* There is no true 'stock' item for timer, so
 			 * instead we draw our own button, and use a
 			 * pair of stock images to toggle between.
 			 */
-			mytbar->timer_button_image = GTK_IMAGE(gtk_image_new());
-			gtk_image_set_from_stock (mytbar->timer_button_image,
-									  GNOME_STOCK_TIMER, GTK_ICON_SIZE_LARGE_TOOLBAR);
-			
-			mytbar->timer_button =
-				gtk_toolbar_append_item(mytbar->tbar,
-										_("Timer"),
-										_("Start/Stop Timer"),
-										NULL,
-										GTK_WIDGET(mytbar->timer_button_image),
-										(GtkSignalFunc) menu_toggle_timer, NULL);
-			position ++;
+			mytbar->timer_button_image = GTK_IMAGE (gtk_image_new ());
+			gtk_image_set_from_stock (
+			    mytbar->timer_button_image, GNOME_STOCK_TIMER, GTK_ICON_SIZE_LARGE_TOOLBAR
+			);
+
+			mytbar->timer_button = gtk_toolbar_append_item (
+			    mytbar->tbar, _ ("Timer"), _ ("Start/Stop Timer"), NULL,
+			    GTK_WIDGET (mytbar->timer_button_image), (GtkSignalFunc) menu_toggle_timer, NULL
+			);
+			position++;
 		}
 		if (config_show_tb_calendar)
 		{
-			mytbar->calendar_w = toolbar_append_stock_button(mytbar->tbar,
-															 _("Calendar"),
-															 _("View Calendar"),
-															 GNOME_STOCK_TEXT_BULLETED_LIST,
-															 (GtkSignalFunc)edit_calendar, NULL);
-			position ++;
+			mytbar->calendar_w = toolbar_append_stock_button (
+			    mytbar->tbar, _ ("Calendar"), _ ("View Calendar"),
+			    GNOME_STOCK_TEXT_BULLETED_LIST, (GtkSignalFunc) edit_calendar, NULL
+			);
+			position++;
 		}
-		if (((config_show_tb_timer)    ||
-			 (config_show_tb_journal)  ||
-			 (config_show_tb_calendar) ||
-			 (config_show_tb_prop)     ) &&
-			((config_show_tb_pref) ||
-			 (config_show_tb_help) ||
-			 (config_show_tb_exit)))
+		if (((config_show_tb_timer) || (config_show_tb_journal) || (config_show_tb_calendar)
+		     || (config_show_tb_prop))
+		    && ((config_show_tb_pref) || (config_show_tb_help) || (config_show_tb_exit)))
```

Comment:
I'm happy to see that it appears to use spaces for alignment and removed the misplaced tabs that were here before. :)